### DISCUSSION TITLE
256x256 PixArt-Sigma model, only get a FID-30k nearly ~49 on coco2014 val , is that normal?

### DISCUSSION THREAD

**Discussion by Gus-Guo** [PixArt-alpha/PixArt-alpha#121] [Q&A]
FID computation is from the codes in https://github.com/mseitzer/pytorch-fid/tree/master. 
30k prompts are radomly sampled from COCO 2014 captions val split.
After generation using these randomly sampled prompts, I compute fid between generated images and images in coco2014 val split.
But the fid result is nearly 49, kind of confused...

**Comment by pratheba** [comment]
I get the around 37.8 using GigaGAN fid computation which the paper mentioned. 
I am using lewei_scale of 1 with the given xl-2 256 model. I am using the ASPECT_RATIO_256_TEST for base_ratios. It would be great if authors can detail how they computed the FID.
